README.md

```diff
@@ -72,6 +72,76 @@ curl https://raw.githubusercontent.com/jaegertracing/jaeger-operator/main/exampl
 
 It is recommended to deploy the operator instead of generating a static manifest.
 
+## Jager V2 Operator
+
+As the Jaeger V2 is released, it is decided that Jaeger V2 will deployed on Kubernetes using [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator). This will benefit both the users of Jaeger and OpenTelemetry. To use Jaeger V2 with OpenTelemetry Operator, the steps are as follows:
+
+* Install the cert-manager in the existing cluster with the command:
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.1/cert-manager.yaml
+```
+
+Please verify all the resources (e.g., Pods and Deployments) are in a ready state in the `cert-manager` namespace.
+
+* Install the OpenTelemetry Operator by running:
+```bash
+kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
```

Comment:
how can we test before the official operator support is released, e.g. from a fork?

Reply:
All the steps mentioned in the readme works right now. You can test all of them by creating a local `Kind` cluster.

Once all the changes are merged and new Operator is released, you won't need to port-forward to a service or a deployment to interact with the UI for any of the extensions which has an endpoint/port defined.

Reply:
I can also demo the steps if you want.